include/dsVideoPort.h

```diff
@@ -660,6 +660,58 @@ dsError_t dsSetForceDisable4KSupport(intptr_t handle, bool disable);
  */
 dsError_t dsGetForceDisable4KSupport(intptr_t handle, bool *disable);
 
+/**
+ * @brief Enables/Disables ALLM mode for HDMI output video port.
+ *
+ * For source devices, this function enables or disables the ALLM mode for specified HDMI output video port.
+ * For sink devices, this function returns dsERR_OPERATION_NOT_SUPPORTED always.
+ *
+ * @param[in] handle    - Handle of the video port returned from dsGetVideoPort()
+ * @param[in] enabled   - Flag to enable/disable the ALLM mode for the HDMI output port
+ *                         ( @a true to enable, @a false to disable)
+ *
+ * @return dsError_t                      -  Status
+ * @retval dsERR_NONE                     -  Success
+ * @retval dsERR_NOT_INITIALIZED          -  Module is not initialised
+ * @retval dsERR_INVALID_PARAM            -  Parameter passed to this function is invalid
+ * @retval dsERR_OPERATION_NOT_SUPPORTED  -  The attempted operation is not supported
+ * @retval dsERR_GENERAL                  -  Underlying undefined platform error
+ *
+ * @pre dsVideoPortInit() and dsGetVideoPort() must be called before calling this API.
```

Comment:
why to call dsGetVideoPort as pre for set call?

Reply:
To get the video port handle